Ginger/GingerCoreCommon/Actions/ActLogAction.cs

```diff
@@ -60,10 +60,20 @@ public string LogText
             }
         }
 
+        public enum LogLevel
```

Comment:
please call the enum "eActionLogLevels" to avoid confusing, also all enums names should start with 'e'

Reply:
Renamed it.